ceph-container-build-ceph-base-push-imgs/config/definitions/ceph-container-build-push-imgs.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: ceph-container-build-ceph-base-push-imgs
+    node: huge && trusty && x86_64
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph-container: build and push ceph base container images to Docker Hub'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 1
+          num-to-keep: 1
```

Comment:
`days-to-keep` and `num-to-keep` can be mutually exclusive: whichever hits first will trigger the removal. If you want a whole day of logs, these settings will remove all except the last one. If that is fine to you, then these would make sense.

Reply:
Yes, I don't even think we want to keep anything, that's just in case ;).